dev-libs/spdlog/files/spdlog-unbundle-fmt.patch

```diff
@@ -1,90 +1,37 @@
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 52d2953..d132d26 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -80,6 +80,7 @@ install(
- install(
-     DIRECTORY "${HEADER_BASE}/${PROJECT_NAME}"
-     DESTINATION "${include_install_dir}"
-+    PATTERN "bundled*" EXCLUDE
- )
- 
- # install project version file
 diff --git a/include/spdlog/fmt/fmt.h b/include/spdlog/fmt/fmt.h
-index 92ca4e5..55f1fa8 100644
+index e2e0485..d33bc45 100644
 --- a/include/spdlog/fmt/fmt.h
 +++ b/include/spdlog/fmt/fmt.h
-@@ -10,25 +10,7 @@
- // By default spdlog include its own copy.
- //
- 
--#if !defined(SPDLOG_FMT_EXTERNAL)
--
--#ifndef FMT_HEADER_ONLY
--#define FMT_HEADER_ONLY
--#endif
--#ifndef FMT_USE_WINDOWS_H
--#define FMT_USE_WINDOWS_H 0
--#endif
+@@ -18,16 +18,11 @@
+ #ifndef FMT_USE_WINDOWS_H
+ #define FMT_USE_WINDOWS_H 0
+ #endif
 -#include "bundled/format.h"
 -#if defined(SPDLOG_FMT_PRINTF)
 -#include "bundled/printf.h"
 -#endif
--
--#else //external fmtlib
--
+ 
+-#else // external fmtlib
++#endif // external fmtlib
+ 
  #include <fmt/format.h>
  #if defined(SPDLOG_FMT_PRINTF)
  #include <fmt/printf.h>
  #endif
--
+ 
 -#endif
--
 diff --git a/include/spdlog/fmt/ostr.h b/include/spdlog/fmt/ostr.h
-index 5cdd5cd..d70c6b3 100644
+index 9902898..5e92a59 100644
 --- a/include/spdlog/fmt/ostr.h
 +++ b/include/spdlog/fmt/ostr.h
-@@ -7,11 +7,6 @@
- 
- // include external or bundled copy of fmtlib's ostream support
- //
--#if !defined(SPDLOG_FMT_EXTERNAL)
--#include "fmt.h"
+@@ -11,8 +11,6 @@
+ #ifndef FMT_HEADER_ONLY
```

Comment:
I'm lost. Don't you have to define `FMT_HEADER_ONLY` for the external case as well, to avoid linking the library?

Reply:
Yes, but why we should remove the possibility to link against fmtlib entirely? Currently by default it uses FMT_HEADER_ONLY and it's not needed to link against fmtlib. If for some reason someone want to link against libmft instead they can use SPDLOG_FMT_EXTERNAL. 
If some package is using SPDLOG_FMT_EXTERNAL explicitly I suppose they will already be linking against fmtlib,  in that case if we make the external case to use FMT_HEADER_ONLY it will both link against fmtlib and compile fmt as header_only.

Reply:
Oh, right. I failed to notice that `SPDLOG_FMT_EXTERNAL` is not defined anymore. Good thinking, great work!